doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -291,6 +301,30 @@ \subsubsection{Zone Total Internal Total Heating Energy {[}J{]}}\label{zone-tota
 
 These output variables represent the sum of all heat gains throughout the zone in Watts (for rate) or joules. This includes all heat gains from People, Lights, Electric Equipment, Gas Equipment, Other Equipment, Hot Water Equipment, and Steam Equipment.
 
+\subsubsection{Zone Exfiltration Heat Transfer Rate {[}W{]}}\label{zone-exfiltration-heat-transfer-rate-w}
+
+\subsubsection{Zone Exfiltration Sensible Heat Transfer Rate {[}W{]}}\label{zone-exfiltration-sensible-heat-transfer-rate-w}
+
+\subsubsection{Zone Exfiltration Latent Heat Transfer Rate {[}W{]}}\label{zone-exfiltration-latent-heat-transfer-rate-w}
+
+These output variables represent the heat emitted to ambient from exfiltration in watts. The exfiltration rate is calculated by solving a mass flow balance on the zone including infiltration, ventilation, outdoor air mixing, zone-to-zone mixing, and all of the zone inlet, exhaust, and return nodes. The latent parts are determined by taking the difference between the total and the sensible rate. The positive values indicate the zone injects heat to the environment, while the negative values indicate the building extracts heat from the environment. (This output variable is only generated if the user has set a computer system environment variable DisplayAdvancedReportVariables equal to "yes".)
```

Comment:
Watts should be capitalized here an below.
~The~ Positive values . . .
while ~the~ negative values . . . <similar changes in other sections below>

Reply:
While it's true the DisplayAdvancedReportVariables can be set as an environment variable, it can also be set in the idf with Output:Diagnostics, DisplayAdvancedReportVariables; In some parts of the I/O Ref Advanced Report Variables are grouped at the end of the list with a statement saying the following are available when advance report variables are requested (ref. Output:Diagnostics, DisplayAdvancedReportVariables):

This raises a question - does the table report work correctly if DisplayAdvancedReportVariables is *not* active? Looking at this now, I would suggest dropping the DisplayAdvancedReportVariables option for heat emissions and just always set them up.

Reply:
I agree with not using advanced report variables here, just always have it on.  It will make the code and docs simpler.

Reply:
I'm removing this but what about https://github.com/NREL/EnergyPlus/blob/452ad34e1c1dd56c687a209ec7fd73d266bb116e/src/EnergyPlus/HeatBalanceSurfaceManager.cc#L1671

should the surface outputs stay when only `DisplayAdvancedReportVariables`, as the other Outside Face vars?

Reply:
Leave those as-is, only touch variables related to heat emissions